modules/olm-enabling-operator-restricted-network.adoc

```diff
@@ -18,18 +19,128 @@ As an Operator author, your Operator must meet additional requirements to run pr
 // TODO: Include more info w/ better steps on how to do this:
 //* You must understand the {product-title} proxy configuration.
 
-For the CSV requirements, you can make the following changes as the Operator author.
+For the CSV requirements, you can make the following changes as the Operator author. The following procedure uses the Memcached Operator created in the Operator SDK tutorials for Go, Ansible, and Helm.
 
 .Prerequisites
 
 * An Operator project with a CSV.
 
 .Procedure
 
-. Use SHA references to related images in two places in the CSV for your Operator:
+. Add the images and tags used by the Operator by setting an environment variable, such as `RELATED_IMAGE_MEMCACHED`, in the `config/manager/manager.yaml` file:
++
+.Example `config/manager/manager.yaml`
+[%collapsible]
+====
+[source,yaml]
+----
+...
+spec:
+  ...
+    spec:
+      ...
+      containers:
+      - command:
+        - /manager
+        ...
+        env:
+        - name: RELATED_IMAGE_MEMCACHED
+          value: "docker.io/memcached@sha256:00b68b00139155817a8b1d69d74865563def06b3af1e6fc79ac541a1b2f6b961"
```

Comment:
You do not need the digest here, just the normal tagged image. It will be resolved to a digest when you run `make bundle`.
```suggestion
          value: "memcached:1.4.36-alpine"
```